C/syntax/SyntaxKind.h

```diff
@@ -409,6 +410,7 @@ enum SyntaxKind : std::uint16_t
     CompoundLiteralExpression,
     CastExpression,
     VAArgumentExpression,
+    ExtGNU_ChooseExpr,
```

Comment:
Should be `ExtGNU_ChooseExpression` for reason similar to that of the first comment.